### PR TITLE
Renew Plugin Admin page

### PR DESCRIPTION
**Pull request by donho**
It's about the first draft for Plugin Admin page.
It is not complete and it is necessary to refine this page with iteration.
The goal of this commit is define the structure of this page, and we can discuss about it (add/remove).

### CODE REVIEW

**Comment by MetaChuh** [comment]
@donho 

thanks for your motivation.

note: @milipili has not yet responded on how we can render the current multi-language structure, as well as the very important previewing of pulls before merging.
so we currently can only review contents and grammar, which we will of course do.

@milipili
it would be nice, if you involve the team while planing and sorting things out, especially if you encounter technical problems.

many thanks and best regards.

File: content/plugins/_index.en.md [hunk 1]
```diff
@@ -3,10 +3,74 @@ title: Plugins
...
 
-When Notepad++ starts, it looks into its Plugins configuration folder and loads whatever dll it finds. Later, you can add plugins using the Settings -> Import -> Import plugins menu. Be careful to make sure your version of Notepad++ is up-to-date enough for new plugins and that the plugin is compatible with Notepad++ (some very old plugins will not work with newer versions of Notepad++, it is up to the developer of the plugin to add support or not).
 
-Currently, Notepad++ can be found in an (older) ANSI version and the newer Unicode version. The plugin has to match this version, otherwise it will not work (Notepad++ will warn you about this). Plugins can be found anywhere on the internet, but a large can be found in the Notepad++ Plugins project on SourceForge.net: http://sourceforge.net/projects/npp-plugins/. There is even a plugin, called Plugin Manager, which works out the installation details and version management for you.
+## How to install a plugin
+The plugin (in the DLL form) should be placed in the \plugins subfolder of the Notepad++ Install Folder, under the subfolder with the same name of plugin binary name without file extension.
```
Comment (pryrt) [inline content/plugins/_index.en.md]:
```suggestion

### Install using Plugins Admin
The Plugins Admin allows you to easily install plugins that are in the Plugins List.  To do so, place a check mark next to the Plugin(s) you wish to install, then select Install.

### Install plugin manually
If the plugin you want to install is not listed in the Plugins Admin, you may still install it manually.  The plugin (in the DLL form) should be placed in the \plugins subfolder of the Notepad++ Install Folder, under the subfolder with the same name of plugin binary name without file extension.
```

**Comment by donho** [comment]
@MetaChuh I believe that we don't really need the part from @milipili to start.
The most important part in a doc book is its structure and content. 
In the worst case we can always use github's jekyll by modifying slightly the structure of this repo if @milipili cannot provide his module in time.
So let's start it and focus on the structure and the content.

File: content/plugins/_index.en.md [hunk 2]
```diff
@@ -3,10 +3,79 @@ title: Plugins
...
+
+Once you installed the plugin, you can use (and you may configure) it via the menu "Plugins".
+
+## How to develop a plugin
+
+### Getting started - Make your plugin in less 10 minutes (C++)
+
+Here are the instructions to make your first Notepad++ plugin in less 10 minutes, by following 6 steps:
+
+1. Download and unzip [Notepad++ Plugin Template](https://github.com/npp-plugins/plugintemplate/archive/v3.zip).
```
Comment (chcg) [inline content/plugins/_index.en.md]: Shouldn't it be https://github.com/npp-plugins/plugintemplate/releases/download/v3.1/pluginTemplate.v3.1.bin.zip and https://github.com/npp-plugins/plugintemplate/releases/download/v3.1/pluginTemplate.v3.1.bin.x64.zip or the outstanding update for n++ 7.7.

File: content/plugins/_index.en.md [hunk 3]
```diff
@@ -3,10 +3,79 @@ title: Plugins
...
+
+However, the knowledge of Notepad++ plugin system is required, if you want to accomplish some sophisticated plugin commands.
+
+You can use [Plugin development forum](https://notepad-plus-plus.org/community/category/5/plugin-development) for any technical questions/answers and the announcement your new plugin.
+
+
+### In other languages
+
+* [Delphi](https://sourceforge.net/projects/npp-plugins/files/DelphiPluginTemplate/DelphiPluginTemplate%202.0%20UNICODE/DelphiPluginTemplate2.zip/download)
+* [C#](http://sourceforge.net/projects/sourcecookifier/files/other%20plugins/NppPlugin.NET.v0.5.zip/download)
```
Comment (chcg) [inline content/plugins/_index.en.md]: Current update therefore is available at https://github.com/kbilsted/NotepadPlusPlusPluginPack.Net

**Comment by donho** [comment]
https://github.com/notepad-plus-plus/npp-usermanual/commit/06c2cc569f3ec9882bcd670fc6953877c526f004

**Pull request closed without merging**